datagateway_api/common/database/backend.py

```diff
@@ -32,6 +37,19 @@ class DatabaseBackend(Backend):
     Class that contains functions to access and modify data in an ICAT database directly
     """
 
+    def ping(self, **kwargs):
+        log.info("Pinging DB connection to ensure API is alive and well")
+
+        try:
+            inspector = inspect(db.engine)
+            print(type(inspector))
```

Comment:
Is this meant to be here? Seems like a debug statement

Reply:
Ah yes, I was working out the path of the function to mock. Removed in https://github.com/ral-facilities/datagateway-api/pull/249/commits/c9027505ec9ebd84923b5aa3bc6c96a82c1a6ae4